docker/compose/docker-compose-wtih-existing-results-db.yml

```diff
@@ -0,0 +1,49 @@
+version: "2"
+services:
+  # Container with mysql base for jaas
+  jaas-mysql:
+    image: mysql:5.7
+    container_name: jaas-mysql
+    ports:
+      - 3308:3306
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_USER: jaggeruser
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: jaasdb
+
+  jagger-web-client:
+    image: griddynamics/jagger-web-client:snapshot
+    container_name: jagger-web-client
+    ports:
+      - 8087:80
+    environment:
+      TZ: Europe/Moscow
```

Comment:
Because in the web UI we will see time in the wrong time zone.  Web-client get time from docker container, by default I think UTC.